examples/translation_example.py

```diff
@@ -64,75 +70,83 @@ def _forward(self, batch):
             max_length=self.config.max_length,
             num_beams=self.config.num_beams,
             num_return_sequences=1,
+            repetition_penalty=1.2,
         )
 
     def forward(self, batch):
-        if self.autocast:
-            with torch.autocast(device_type="cuda"):
-                outputs = self._forward(batch)
-        else:
-            outputs = self._forward(batch)
```

Comment:
Why was this removed ?

Reply:
Actually this was done to make nearly exact replica of that standalone script, so that we can measure the improvement only from CrossFit addition. So I had to remove this since it was not there in the standalone script